editor.planx.uk/src/components/EditorNavMenu.tsx

```diff
@@ -73,32 +73,33 @@ const TooltipWrap = styled(({ className, ...props }: TooltipProps) => (
 }));
 
 const MenuButton = styled(IconButton, {
-  shouldForwardProp: (prop) =>
-    !["isActive", "disabled"].includes(prop as string),
-})<{ isActive: boolean; disabled?: boolean }>(
-  ({ theme, isActive, disabled }) => ({
+  shouldForwardProp: (prop) => prop !== "isActive",
+})<{ isActive: boolean }>(({ theme, isActive }) => ({
```

Comment:
As `disabled` is an `IconButton` prop, you should still be able to access it here.

```ts
})<{ isActive: boolean }>(({ theme, isActive, disabled }) => ({
```

Reply:
Ah yes, I've updated, allows a consistent use of the prop with the CSS